fetch.bs

```diff
@@ -3941,7 +3946,7 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
 
   <ol>
    <li>
-    <p>While true, breaking if the ongoing fetch <a for=fetch lt=terminated>terminates</a>:
+    <p>While true, <a>abort when</a> the ongoing fetch <a for=fetch lt=terminated>terminates</a>:
```

Comment:
This is the only one I'm unsure of, as it feels like the "abort" clause is inside the while clause, whereas it's really the other way around.

Reply:
That's a good point, I guess I should embrace the additional nested step you suggested.